Bottom: 45bd06c7a31a8dcb66e9ed929e6531221f9c4ca2
Top:    ff333b471b44effa39366da627b85b5629d1ebbc
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-11-21 21:56:05 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
